pay/GDPR/GDPR_CZ_cs.md

```diff
@@ -1,6 +1,8 @@
 **Zásady ochrany soukromí v Trisbee**
 ======================================
 
+## **Zásady ochrany soukromí a nařízení GDPR**
```

Comment:
Oh I see.
Let's just try to replace: `**Zásady ochrany soukromí v Trisbee**` by: `# Zásady ochrany soukromí v Trisbee`.